sqlmesh/cli/example_project.py

```diff
@@ -8,7 +8,7 @@
     local:
         connection:
             type: duckdb
-            database: db.db
+            database: ./db.db
```

Comment:
why did you change this

Reply:
The quickstart db currently writes to the __file__ location, which I found unintuitive as a user (and is unstable depending on python config).

I noticed it while repeatedly deleting and re-initializing the project.

Should I revert?

Reply:
so it writes to the same location as the project?

Reply:
yes, in the project directory - db.db and the wal

Reply:
logic being that we can be sure the directory where `init` is called is writable and easily located by the user

Reply:
to me, db.db should be colocated with the example project, not where you run it, is that the current behavior?

Reply:
By colocated, do you mean in the same directory as the example project directory, or do you mean inside the example project directory where config.yaml is located?

Reply:
Either way, i'm having trouble consistently reproducing the behavior in my dev environment in a fresh dir, venv, and pip. We haven't heard any complaints about this, so i'll go ahead and revert.